lib/at_host/at_host.c

```diff
@@ -23,21 +23,43 @@ LOG_MODULE_REGISTER(LOG_DOMAIN);
 #define INVALID_DESCRIPTOR -1
 
 #define UART_RX_BUF_SIZE 4096 /**< UART RX buffer size. */
+
+#define THREAD_STACK_SIZE (UART_RX_BUF_SIZE + 1024)
+#define THREAD_PRIORITY 5
```

Comment:
Not a big deal as the outcome is the same, but using `K_PRIO_PREEMPT(5)` improves readability a bit IMHO, as we clearly see what type of thread we deal with.